f2/src/App.js

```diff
@@ -52,9 +53,14 @@ const App = () => {
             bg="gray.50"
           >
             <Home />
-          </Flex>
+            <MidFeedbackForm
+              onSubmit={data => {
+                console.log(data)
```

Comment:
I don't know why, but this isn't logging anything for me.

Reply:
Its temporary. I should have commented that. Did not want to create a postData function since you had already done the same in #1360